model-providers/ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/runtime/config/LangChain4jOllamaConfig.java

```diff
@@ -69,6 +69,16 @@ interface OllamaConfig {
         @WithDefault("true")
         Boolean enableIntegration();
 
+        /**
+         * the experimental tools name. Currently, the only accepted values are {@code NONE, PARALLEL, SEQUENTIAL}
+         * NONE: for no tools, works as before without handling tools.
+         * PARALLEL: Tools we be used and simulated with one call to llm, that will answer with all tool request to execute
+         * and the response using the result of the tool request.
+         * SEQUENTIAL: Tools will be call sequentially and llm will call next tool following tool request result.
+         */
+        @WithDefault("NONE")
+        Optional<String> experimentalTools();
```

Comment:
I would just keep `NONE` and whatever works best in all cases. I  don't see the reason for providing a option for serial and parallel

Reply:
agreed!

Reply:
done